### PR TITLE
wrapper: Expose getACLTransactionPath

### PR DESCRIPTION
**Pull request by izqui**
Exposes a function that returns the transaction path for ACL actions.

### CODE REVIEW

File: packages/aragon-wrapper/src/index.js [hunk 1]
```diff
@@ -523,60 +523,8 @@ export default class Aragon {
...
-   *
-   * @param {string} appAddress
-   * @param {string} roleHash
-   * @return {Promise<string>} The permission manager
-   */
-  async getPermissionManager (appAddress, roleHash) {
-    const permissions = await this.permissions.take(1).toPromise()
-
-    return dotprop.get(permissions, `${appAddress}.${roleHash}.manager`)
+    const path = await this.calculareACLTransactionPath(method, params)
```
Comment (sohkai) [inline packages/aragon-wrapper/src/index.js]: Typo: should be `this.getACLTransactionPath `

File: packages/aragon-wrapper/src/index.js [hunk 2]
```diff
@@ -523,60 +523,8 @@ export default class Aragon {
...
-    if (!functionArtifact) {
-      throw new Error(`Method ${method} not found on ACL artifact`)
-    }
-
-    if (functionArtifact.roles && functionArtifact.roles.length !== 0) {
-      // createPermission can be done with regular transaction pathing (it has a regular ACL role)
-      const path = await this.getTransactionPath(aclAddr, method, params)
-      return this.performTransactionPath(path)
-    } else {
-      // All other ACL functions don't have a role, the manager needs to be provided to aid transaciton pathing
```
Comment (Quazia) [inline packages/aragon-wrapper/src/index.js]: typo: transaciton -> transaction

File: packages/aragon-wrapper/src/index.js [hunk 3]
```diff
@@ -668,6 +616,68 @@ export default class Aragon {
...
+
+    if (!functionArtifact) {
+      throw new Error(`Method ${method} not found on ACL artifact`)
+    }
+
+    if (functionArtifact.roles && functionArtifact.roles.length !== 0) {
+      // createPermission can be done with regular transaction pathing (it has a regular ACL role)
+      return this.getTransactionPath(aclAddr, method, params)
+    } else {
+      // All other ACL functions don't have a role, the manager needs to be provided to aid transaciton pathing
```
Comment (Quazia) [inline packages/aragon-wrapper/src/index.js]: typo: transaciton -> transaction

**Comment by Quazia** [comment]
LGTM

**Pull request merged**